pkg/aliases/script/docker.go

```diff
@@ -145,10 +145,11 @@ func (adpt *DockerRunAdapter) Command(client *docker.Client, overrideArgs []stri
 	if len(overrideArgs) > 0 {
 		args = overrideArgs
 	}
-	opt := overrideOption
-	if err := mergo.Merge(&opt, adpt.Option(), mergo.WithAppendSlice); err != nil {
+	dst := overrideOption
+	if err := mergo.Merge(&dst, *adpt.Option(), mergo.WithAppendSlice); err != nil {
```

Comment:
https://github.com/imdario/mergo#transformers
The case of overwriting with an empty string cannot be solved by mergo transformers.

I can not understand that mergo calls `!isEmptyValue(dst)` to use the transformer.
Therefore, I can not fix mergo.